hw/ip/rom_ctrl/dv/rom_ctrl_sim.core

```diff
@@ -0,0 +1,30 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:rom_ctrl_sim:0.1"
+description: "ROM_CTRL DV sim target"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:rom_ctrl
+
+  files_dv:
+    depend:
+      - lowrisc:dv:rom_ctrl_test
+      - lowrisc:dv:rom_ctrl_sva
+    files:
+      - tb.sv
+    file_type: systemVerilogSource
+
+targets:
+  sim: &sim_target
+    toplevel: tb
+    filesets:
+      - files_rtl
+      - files_dv
+    default_tool: vcs
+
+  # TODO: add a lint check cfg in `hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson`
```

Comment:
Would you please address this TODO as well?

Reply:
Will do